update_mapping.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+import json
+import re
+import boto3
+from lib import env
+from datetime import date, datetime
+
+#This script create a json 
```

Comment:
What does this do?

Reply:
@emephield 24 days later. What those this do ?
We talked about it in scrum. no updates...